isis/tests/FunctionalTestsApollocal.cpp

```diff
@@ -1,8 +1,9 @@
+#include "Brick.h"
 #include "Fixtures.h"
 #include "Pvl.h"
 #include "PvlGroup.h"
 #include "TestUtilities.h"
-#include "Histogram.h"
+#include "Statistics.h"
```

Comment:
Why the switch? Was this due to the change that we made to histogram vs statistics?

Reply:
The Cube function gets a Histogram of the whole cube. We don't need that much, so I'm just using a Statistics object and manually adding the data from the bricks to it.